src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetFiltersMappingPage.tsx

```diff
@@ -315,7 +315,7 @@ export default function ReleaseApiDataSetFiltersMappingPage() {
                   className="govuk-heading-l dfe-flex dfe-align-items--center"
                   id="mappable-filter-columns"
                 >
-                  Filter columns not found in the new data set{' '}
+                  Filter columns not found in new data set{' '}
```

Comment:
This is entirely subjective, but I think this, and the other places you've changed it, read better with "the".